pkg/netlink/netlink.go

```diff
@@ -92,6 +93,11 @@ var (
 		k8snet.IPv4: allZeroAddress,
 		k8snet.IPv6: allZeroAddressV6,
 	}
+
+	IPFamilyToNetlinkFamily = map[k8snet.IPFamily]int{
+		k8snet.IPv4: unix.AF_INET,
+		k8snet.IPv6: unix.AF_INET6,
+	}
```

Comment:
There is an existing `ToNetlinkFamily` function. We should use either one or the other - the map here is simpler.